pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/extension/PythonVersion.java

```diff
@@ -25,25 +25,43 @@ public PythonVersion(String version) {
         this.version = version;
     }
 
+    private String getVersionPart(int position) {
+        String[] parts = this.version.split("\\.");
+        return parts[position];
+    }
+
     /**
      * @return The exact version of Python this project uses, such as '2.7.11'.
      */
     public String getPythonVersion() {
-        return version;
+        return this.version;
```

Comment:
No need for `this` here. There's no ambiguity with the name as it happens in the setter or constructor where the parameter is also named `version`. Here, in the getter, it can only be our internal, private field `version`. Let's leave this method unchanged.